time-y1-nozzle.sh

```diff
@@ -3,79 +3,162 @@
 set -e
 set -x
 
+TIMING_HOME=$(pwd)
+TIMING_HOST=$(hostname)
+TIMING_DATE=$(date +"%m-%d-%y")
```

Comment:
As resident Eurotrash, hate hate hate. :)

Compromise?
```suggestion
TIMING_DATE=$(date "+%Y-%m-%d %H:%M")
```
i.e. `2021-03-03 09:21`